program-runtime/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 #![cfg_attr(RUSTC_WITH_SPECIALIZATION, feature(min_specialization))]
-#![allow(clippy::integer_arithmetic)]
+#![allow(clippy::integer_arithmetic)] // TODO: Remove
```

Comment:
Still needs to be removed?

Reply:
Yes, I have another branch for a follow-up PR on this one, which is going to move everything from `sdk::process_instruction` into the program-runtime-crate. And it will remove that clippy flag.

https://github.com/Lichtso/solana/tree/refactor/move_process_instruction_in_program_runtime